public/templates/header.php

```diff
@@ -40,8 +40,12 @@
                 <li><a class="dropdown-item" href="#">Course 1 </a></li>
                 <li><a class="dropdown-item" href="#">Direct Messages</a></li>
                 <li><a class="dropdown-item" href="#">Teams</a></li>
+<<<<<<< HEAD
+                <li><a class="dropdown-item" href="meetings.php">Meetings</a></li> 
+=======
                 <li><a class="dropdown-item" href="#">Meetings</a></li> 
                 <li><a class="dropdown-item" href="marked_entities.php">Marked Entities</a></li>
+>>>>>>> main
```

Comment:
:o

Reply:
this is not on my file on my computer, I fetched form the github, but its still not there, I'm confused